src/pages/matchList.tsx

```diff
@@ -25,114 +33,122 @@ type Match = {
-              direction="column"
-              gap="sm"
-              key={coat}
-              style={{ backgroundColor: "#e0f0e0", borderRadius: "0.5rem" }}
-            >
-              <Title order={3}>{parseInt(coat) + 1}コート</Title>
-              {matches.map((match) => {
-                return (
-                  <MatchCard
-                    key={match.id}
-                    id={match.id}
-                    matchType={match.matchType}
-                    teams={match.teams}
-                    isFinished={match.results != null}
-                  />
-                );
-              })}
-            </Flex>
-          ))
-        ) : (
-          <Title w={"100%"} order={3}>
-            まだ決勝戦の組み合わせは決まっていません
-          </Title>
-        )}
-      </Flex>
-    </Box>
+    <>
+      <Title order={1} m="1rem">
+        試合表
+      </Title>
+      {primaryMatches ? (
+        <>
+          <Flex justify="flex-end" mb={"1rem"}>
+            <CourseSelector courses={courses} selector={setSelect} />
+          </Flex>
+          <Table highlightOnHover>
+            <Table.Thead>
+              <Table.Tr>
+                <Table.Th>コート番号</Table.Th>
+                <Table.Th>左コート</Table.Th>
+                <Table.Th>右コート</Table.Th>
+                <Table.Th>
+                  <Center>状態</Center>
+                </Table.Th>
+              </Table.Tr>
+            </Table.Thead>
+            <Table.Tbody>
+              {primaryMatches.map(
+                (match) =>
+                  (select === "all" || match.courseIndex === select) && (
+                    <Table.Tr key={match.id}>
+                      <Table.Td>
+                        <Center miw={50}>
+                          <Text fw={700}>{match.courseIndex}</Text>
+                        </Center>
+                      </Table.Td>
+                      <Table.Td>
+                        <Text fw={700} miw={200} ta={"start"}>
+                          {match.teams.left.teamName}
+                        </Text>
+                      </Table.Td>
+                      <Table.Td>
+                        <Text fw={700} miw={200} ta={"start"}>
+                          {match.teams.right.teamName}
+                        </Text>
+                      </Table.Td>
+                      <Table.Td>
+                        <Center>
+                          <MatchStatusButton
+                            status={match.results ? "end" : "future"}
+                            id={match.id}
+                            teams={match.teams}
+                            matchType={match.matchType}
+                          />
+                        </Center>
+                      </Table.Td>
+                    </Table.Tr>
+                  )
+              )}
+            </Table.Tbody>
+          </Table>
+        </>
+      ) : loading ? (
+        <>
+          <Text>ロード中</Text>
+          <Loader size={40} />
+        </>
+      ) : error ? (
+        <>
```

Comment:
三項演算子のネストは読みづらいので何か別の方法を考えたほうがいいかもしれません。
```suggestion
        </>
      )}
      {loading && (
        <>
          <Text>ロード中</Text>
          <Loader size={40} />
        </>
      )}
      {error && (
        <>
```
↑のほうがまだいいでしょうか？(他の人の意見も聞きたいです)